python/src/aac/plugins/validators/_validator_result.py

```diff
@@ -37,4 +37,13 @@ def format_message(finding: ValidatorFinding) -> str:
                 f"at {loc.location.line + 1}:{loc.location.column} {finding.message}"
             )
 
-        return "\n".join([format_message(finding) for finding in self.findings.get_all_findings()])
+        if len(self.findings.get_error_findings()) == 0:
```

Comment:
We shouldn't replicate the check, here:

(See below for reasoning related to `valid_files_output`)

```suggestion
        valid_files_output = ""
        if self.is_valid():
```